integration-tests/README.md

```diff
@@ -53,13 +53,13 @@ You can use this flag to test in a remote machine too.
 With the --filter flag you can select the tests to run. For instance you can
 pass MyTestSuite, MyTestSuite.FirstCustomTest or MyTestSuite.*CustomTest:
```

Comment:
If you quote that with backticks, it'll properly set it as code, and it'll also escape the \* which is messing things up right now. If you don't want to backtick-quote, add a \\ before the \*.